### PR TITLE
clean up dup packages and use yarn as client

### PR DESCRIPTION
**Pull request by jquense**
Not sure if this is working yet :P lets see what CI says

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 1738f91584cc3047b07ac8afeeddc8471b8ff418

https://app.netlify.com/sites/using-glamor/deploys/59a438a20752d044763113dd

**Comment by jquense** [comment]
I think this is good to go now. Should solve some annoying bootstrap and hoisting issues

File: packages/gatsby-source-mongodb/package.json.lerna_backup [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (KyleAMathews) [inline packages/gatsby-source-mongodb/package.json.lerna_backup]: I'll assume these lerna_backup files got added by accident. I'll delete them and then merge

**Comment by KyleAMathews** [comment]
Ooo this is so much nicer. Ran it first time and it finished in ~180 seconds. The second time it only took ~30 seconds!!

**Comment by KyleAMathews** [comment]
Haha and it means republishing about every package. If there was a scoreboard for NPM publishes/day, I'd have to be near the top.

**Pull request merged**